tests/data/format/final_period/function_docstrings.py

```diff
@@ -0,0 +1,29 @@
+def func()
+    """Docstring"""
+
+    def inner_func()
+        """Docstring"""
+
+
+def func()
+    """A multi-line
+    docstring
+    """
+
+    def inner_func()
+        """A multi-line
+        docstring
+        """
+
```

Comment:
```suggestion

    def other_inner():
        """A docstring with a sentence ending with dot. And another one
        continuing later."""
```

I think the result should be:
```python
    def other_inner():
        """A docstring with a sentence ending with dot.

        And another one continuing later."""
```

Reply:
I didn't want to do too much with line wrapping and breaking in this and only focus on adding periods when we 99% know that they should be there (I skipped the bodies of multi line strings for the same reason). But I agree this should be done at some point.

What about `SplitSummaryAndDocstringFormatter` which splits on periods? That should be relatively easy to implement.

Reply:
https://github.com/DanielNoord/pydocstringformatter/issues/25